doc/en/user/source/community/ogc-api/features/index.rst

```diff
@@ -167,11 +167,11 @@ To override an OGC API Features template:
 
 #. Create a file in this location, using the GeoServer |release| examples below:
 
-   * :download:`ogc/features/collection.ftl  </../../../../src/community/ogcapi/ogcapi-features/src/main/resources/org/geoserver/ogcapi/features/collection.ftl>`
-   * :download:`ogc/features/collection_include.ftl  </../../../../src/community/ogcapi/ogcapi-features/src/main/resources/org/geoserver/ogcapi/features/collection_include.ftl>`
-   * :download:`ogc/features/collections.ftl  </../../../../src/community/ogcapi/ogcapi-features/src/main/resources/org/geoserver/ogcapi/features/collections.ftl>`
+   * :download:`ogc/features/collection.ftl  </../../../../src/community/ogcapi/ogcapi-features/src/main/resources/org/geoserver/ogcapi/v1/features/collection.ftl>`
```

Comment:
This file looks like its part of a different change.

Reply:
It was indeed a different PR that already got merged. Let me rebase.

Reply:
And just like that... pooof! gone!